azure_sdk/lib/latest/modules/analysisservices_profile_module.rb

```diff
@@ -39,12 +39,18 @@ def initialize(configurable, base_url=nil, options=nil)
           if(client_0.respond_to?(:subscription_id))
             client_0.subscription_id = configurable.subscription_id
           end
-
+          add_telemetry(client_0)
           @servers = client_0.servers
 
           @model_classes = ModelClasses.new
         end
 
+        def add_telemetry(client)
+          profile_information = 'Profiles/Rollup/Mgmt/AnalysisServices'
```

Comment:
is there a reason for us not to follow same order (similar string) as in the namespace, for example record.. "Azure/Profiles/Latest/AnalysisServices/Mgmt"
Does the full telemetry string end up looking something like this: "Profiles/Rollup/Mgmt/AnalysisServices/Latest/azure_mgmt_analysis_services/0,15,2" ? Are we able to infer the api-versions from this user agent? In the case of the person using a profile, I think we could by checking the configuration of the "Latest" profile, in the case of using a specific api-version, would we be able to detect which api-version is being used? I'm trying to look at this overall, so we know telemetry gives us what we need.

Reply:
Done. I have changed the format. The version is always available from the additional string. So, we do not have to add them here too

Reply:
I believe you're referring to the version of the gem, I was wondering about the api-version and if what we get overall would be enough for what we want. Thoughts? For profiles itself seems enough, though I believe we'd need to look at the profile configuration for that gem version to know which api-version got picked up, so was wondering if we wanted to consider adding something else.